tests/unit/test_user.py

```diff
@@ -26,6 +26,37 @@ def test_full_name(empty_test_db, create_admin_user):
     assert admin.full_name() == "first last"
 
 
+@pytest.mark.usefixtures("empty_test_db")
+class TestFindUsersWithoutAssignedRole:
+    def test_find_users_authorized_active(self, create_admin_user):
+        _ = create_admin_user()
```

Comment:
Do we need to assign this? If we're not going to use the assignment, can we simply do `create_admin_user()`?